api/services/call_log_event_service.py

```diff
@@ -25,4 +27,10 @@ def create_call_log_event(self, data):
             if data.get("log_created_on", None)
             else datetime.now(),
         )
-        helpers.save(call_log_event)
+        try:
+            helpers.save(call_log_event)
+            logger.info("Call Log Event saved successfully.")
+        except Exception as e:
+            logger.error("Error occurred while saving Call Log Event: " + str(e))
```

Comment:
Lets use formatted string. It looks more clean
```suggestion
            logger.error(f"Error occurred while saving Call Log Event: {e}")
```

Reply:
Also add phone number of user or anything that will help us to know for which the error occured

Reply:
Updated